ppr-ui/src/components/dialogs/SecuredPartyDialog.vue

```diff
@@ -151,11 +188,25 @@ export default defineComponent({
     defaultResults: {
       type: Array as () => Array<SearchPartyIF>
     },
+    isDuplicate: {
+      type: Boolean,
+      default: false
+    },
+    isReview: {
+      type: Boolean,
+      default: false
+    },
     defaultIsRegisteringParty: Boolean
   },
   setup (props, context) {
     const localState = reactive({
       showSelected: true,
+      duplicate: computed((): boolean => {
+        return props.isDuplicate
+      }),
+      review: computed((): boolean => {
+        return props.isReview
+      }),
```

Comment:
Not really necessary. I would remove the local computed values here and use the props. 
It also helps improve the naming convention, `isDuplicate` and `isReview` is a little more intuitive/representative of a boolean. 

I've seen this implementation of a few places where the props are essentially aliased through a computed property and it's just not required. The prop can be used directly in the template and will be reactive. In some instances i've even seen the alias'ing the cause of some properties becoming Non Reactive.

Reply:
Done, good catch. Getting more familiar with how/when to use props vs localState in the template

Reply:
You are not alone! 
The script syntax changed a between Vue versions and this little bit was just part of the learning curve from that version jump. 
Thank you for the clean up 👍